app/src/main/java/org/mozilla/fenix/components/metrics/GleanMetricsService.kt

```diff
@@ -495,7 +496,7 @@ class GleanMetricsService(private val context: Context) : MetricsService {
         }
         // setStartupMetrics is not a fast function. It does not need to be done before we can consider
         // ourselves initialized. So, let's do it, well, later.
-        gleanSetStartupMetrics = MainScope().launch {
+        gleanSetStartupMetrics = MainScope().launch(IO) {
             setStartupMetrics()
```

Comment:
This doesn't remove the redundant computation in `setStartupMetrics`, wasting user's battery and clogging up our worker threads.